tests/unit/DateTime/DifferenceTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Coduo\PHPHumanizer\Tests\DateTime;
+
+use PHPUnit_Framework_TestCase;
+
+/**
+ * @author       Ezra Pool <ezra@tsdme.nl>
+ */
```

Comment:
If its not a problem for you please remove those doc blocks. Not because I dont like it, we are just not using docblocks to mention authors, everything is in git log and after some time is hard to keep dockblocks and git log synchronized :P